pkg/flow/grpc-mirror.go

```diff
@@ -841,7 +841,7 @@ resend:
 
 func (flow *flow) CancelMirrorActivity(ctx context.Context, req *grpc.CancelMirrorActivityRequest) (*emptypb.Empty, error) {
 	flow.sugar.Debugf("Handling gRPC request: %s", this())
-
+	flow.logger.Debugf(ctx, flow.ID, flow.GetAttributes(), "cancelled by api request")
```

Comment:
Perhaps call this log from the `cancelActivity` function and pass in the same argument provided to it?

Reply:
canceling an action is logged already somewhere in the error-handling, i think...
Its even possible that this log line is a unnecessary duplicate. 
Anyway it would be challenging to pass ctx into cancelActivity